### PR TITLE
Add drop off markers

### PR DESCRIPTION
**Pull request by weijeffrey**
Added sets of drop off markers.
Issue with $on, $emit and $broadcast.
I'm using it to communicate between HomeCtrl and PickUpCtrl.
As valet when I accept the order(in HomeCtrl), the order info is being sent to the (PickUpCtrl) 
This somehow stopped working, but if I refresh while I'm on Pick up page and go back to home, the communication between controllers works.

Spent hours fixing, but got no where.

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 angular.module('starter.controllers', [])
 
-.controller('AppCtrl', function($scope, $ionicModal, $timeout, $auth, $ionicPopup, $window, $log, $state, $http) {
+.controller('AppCtrl', function($scope, $ionicModal, $timeout, $auth, $ionicPopup, $window, $log, $state, $http, $rootScope) {
 
-  $scope.rootURL = "http://localhost:3000/"  
+  $scope.rootURL = "http://localhost:3000/"
```
Comment (f3r) [inline www/js/controllers.js]: would this work in heroku?

File: www/js/controllers.js [hunk 2]
```diff
@@ -226,52 +282,404 @@ angular.module('starter.controllers', [])
...
+            $scope.allMarkers.push(newMarker)
+
+
+          }
+        console.log($scope.allMarkers)
+        })
+      }
+
+      $http.get($scope.rootURL + "api/v1/requests/pickup").success(function(indexPickups){
+      for (var num = 0; num < indexPickups.length; num++){
```
Comment (f3r) [inline www/js/controllers.js:326]: indentation! :cry:

**Comment by h8rry** [comment]
@f3r this is not done yet

**Comment by weijeffrey** [comment]
yep restructuring in progress. will shoot something over soon

thanks

File: www/js/controllers.js [hunk 3]
```diff
@@ -226,52 +282,404 @@ angular.module('starter.controllers', [])
...
+          destinations: [myDestination],
+          travelMode : google.maps.TravelMode.DRIVING
+        }
+         
+        matrixService.getDistanceMatrix(durationDistance, function(responseD, status){
+          if (status == google.maps.DistanceMatrixStatus.OK){
+            distanceEST = responseD.rows[0].elements[0].distance.text;
+            timeEST     = responseD.rows[0].elements[0].duration.text;
+
+            if (transmission == true){
```
Comment (f3r) [inline www/js/controllers.js]:
``` js
transmission = transmission ? 'manual' : 'automatic';
```

File: www/js/services/privatePubServices.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+app.service('PrivatePubServices', function() {
+  var x = {};
+  x.subscribe = function (channel) {
+    PrivatePub.sign({
+      channel: channel,
+      timestamp: 1302306682972,
+      signature: "dc1c71d3e959ebb6f49aa6af0c86304a0740088d",
+      server: "http://localhost:9292/faye"
```
Comment (f3r) [inline www/js/services/privatePubServices.js]: this here may break when deploying to heroku

**Comment by f3r** [comment]
@weijeffrey can you ping me up when this is ready for review?

**Pull request merged**